st2tests/st2tests/base.py

```diff
@@ -465,6 +469,69 @@ def setUp(self):
             self._register_pack_configs()
 
 
+class InstallFixturesPacks(DbTestCase):
```

Comment:
Please look for a simpler alternative first thru the environment setup in circleci.  May be simpler than writing a brand new class to do pack and venv setup here.